CardMaker/Card/Translation/JavascriptHost.cs

```diff
@@ -65,5 +95,25 @@ public void AddOverrideField(string sField, string sValue)
             }
         }
 
+        protected virtual void Dispose(bool disposing)
+        {
+            if (!disposedValue)
+            {
+                if (disposing)
+                {
+                    ScriptEngine.Dispose();
+                }
+
+                disposedValue = true;
+            }
+        }
+
+        [NoScriptAccess]
+        public void Dispose()
```

Comment:
I don't know. The Engine was already in a `using` block, and since I moved it into the Host object (mostly so one function could return both more easily,) I figure if I'm putting a Disposable thing in a member variable I should probably make the host class Disposable too just to be safe.